packages/flutter/test/gestures/gesture_tester.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
+import 'package:meta/meta.dart';
```

Comment:
i'm surprised this didn't cause a test failure... i thought we had a script that caught uses of meta/meta.dart. Maybe it's limited to only the flutter/lib/src directory?

Reply:
From looking, it looks like that script is just checking in some combination of `flutter/lib/` and `flutter/lib/src/`.

Is there an issue w/ importing `package:meta`? We want he `isTest` annotation, which isn't currently exported from foundation (and, the annotation isn't something users will generally need to use).

Reply:
No issue. We check it in the framework because we wanted to reduce the surface area that depended explicitly on other packages so that we could easily adjust that. I was just surprised that the tests weren't included in that.